### ISSUE TITLE
client.get_providers_of() produces error

### ISSUE THREAD

**Issue by ezrakahn** [GreenDelta/olca-ipc.py#13]
I am following along the example in ipc.py for get_producers_of:

```
refid = 'b7127e3e-f5a7-307e-8072-30378b9ed1c2'
flow = client.get('Flow', refid)

for provider in client.get_providers_of(flow):
    print(provider.name)
```
And this trips the err:
`
ERROR:root:failed to get providers: -32601: Does not understand: get/providers
`

This is using the latest build on master:
https://github.com/GreenDelta/olca-ipc.py/commit/371428d733be5ccb18b407b47722b8aaeee85e83

The flow in question is:
https://www.lcacommons.gov/lca-collaboration/National_Renewable_Energy_Laboratory/USLCI_2021_Q1_v1/dataset/FLOW/b7127e3e-f5a7-307e-8072-30378b9ed1c2

**Comment by msrocka** [comment]
This function is only available in openLCA 2.0 (which is not released yet). (Unstable) Development builds where this function should work are available in [this shared folder](https://share.greendelta.com/index.php/s/2AJJMhfB1WSIFDO) (but check the readme-file in that folder).

**Comment by ezrakahn** [comment]
thanks!  Looking forward to the new release!
